Bottom: 83c2ff86feab1cad6d86ae4d207f185abafe3e39
Top:    c9c3a4875d85212bfaca7b08ffb7d88c538fbc71
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-07 18:07:46 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index d387f43..0b03204 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
